nupic/torch/modules/k_winners.py

```diff
@@ -259,7 +304,10 @@ def __init__(
         boost_strength=1.0,
         boost_strength_factor=0.9,
         duty_cycle_period=1000,
-        local=False
+        local=False,
+        break_ties=True,
```

Comment:
Maybe we should default to break_ties=False assuming it is unlikely it will impact the accuracy on most common use cases (float32)